app/scripts/controllers/SetUsernameCtrl.js

```diff
@@ -0,0 +1,19 @@
+(function() {
+    function SetUsernameCtrl($cookies, $uibModalInstance) { //need to inject cookie service?
+        this.setUsername = function(newUser) {  //function to add user to cookie
+            $cookies.put('blocChatCurrentUser', 'newUser');  //replace current key with new one
```

Comment:
'newUser' will be set as the value here. should be `newUser` so it uses the variable it was passed in if you'd like to go this route